src/app/d2-vendors/vendor-ratings.ts

```diff
@@ -0,0 +1,41 @@
+import { DestinyTrackerServiceType } from "../item-review/destiny-tracker.service";
+import { DestinyVendorsResponse, DestinyVendorSaleItemComponent, DestinyVendorResponse, DestinyProfileResponse, DestinyVendorItemDefinition } from "bungie-api-ts/destiny2";
+import * as _ from "underscore";
+import { D2ManifestDefinitions } from "../destiny2/d2-definitions.service";
+
+export async function fetchRatings(destinyTrackerService: DestinyTrackerServiceType,
+                                   vendorsResponse?: DestinyVendorsResponse,
+                                   vendorResponse?: DestinyVendorResponse,
+                                   profileResponse?: DestinyProfileResponse,
+                                   defs?: D2ManifestDefinitions): Promise<void> {
```

Comment:
Should this return its data, instead of updating something else?

Reply:
It initially was returning the cache. Now it's just using the Promise to signal that the bulk fetch is done. Rating pulls are coming out of our local cache (and ratings will too, if they're in there).